docs/src/model_docs/shared_concepts.md

```diff
@@ -1,8 +1,6 @@
-# Vertical processes
+# Shared model components
```

Comment:
I think `processes` is better here, these are not really components of a Model.

Reply:
That's true, changed the header to `Shared processes`. I noticed that we currently describe the rainfall interception processes here as well, while they are in fact only used by the `sbm` concept, right?

Reply:
Yes, indeed, so better to move/merge the rainfall interception part with the SBM section.

Reply:
The interception section is now moved to (and merged with) the SBM section.